bin/configure-tower-projects.py

```diff
@@ -365,37 +382,54 @@ def create(self) -> dict:
         response = self.tower.request("POST", endpoint, json=data)
         return response["workspace"]
 
-    def add_participant(self, user: str, role: str) -> int:
+    def add_participant(self, role: str, user: str = None, team_id: int = None) -> int:
         """Add user to the workspace (if need be) and return participant ID
```

Comment:
Nit: This function now adds both participant and team? Ok with function name but improve docs?

Reply:
This isn't a nit. The docs were inaccurate, so I've updated them. "Participant" is the term that Tower uses to refer to users or teams added to a workspace.

Reply:
Agreed, the docstring for this function still seems unchanged.

Reply:
Sorry, I forgot to upload my commit.